photon-lib/src/main/java/org/photonvision/PhotonCamera.java

```diff
@@ -43,9 +43,17 @@
 
 /** Represents a camera that is connected to PhotonVision. */
 public class PhotonCamera {
-    protected final NetworkTable rootTable;
+    protected final NetworkTable cameraTable;
+
+    // NetworkTables entries
+    // Used
     RawSubscriber rawBytesEntry;
```

Comment:
Why is this public? Is the purpose to expose the entries to the user for them to use in their own code?

Reply:
They are not public, they're protected? It's so SimPhotonCamera can access them.

Reply:
Java sets those to public when not explicitly stated unless I am remembering incorrectly

Reply:
Nope. Default is package private.